src/graphql/types/scalar/ln-pubkey.ts

```diff
@@ -14,12 +18,12 @@ const LnPubkey = GT.Scalar({
   },
 })
 
-function validLnPubkey(value) {
-  // TODO: verify/improve
-  if (value.match(/^[a-f0-9]{66}$/i)) {
-    return value
+function validLnPubkey(value: string): Pubkey | InputValidationError {
+  const pubkey = checkedToPubkey(value)
+  if (pubkey instanceof Error) {
+    return new InputValidationError({ message: "Invalid value for LnPubkey" })
```

Comment:
Shouldn't this be an OutputValidationError?

Reply:
`parseValue` and `parseLiteral` are always inputs. 
the output are managed by `serialize`

https://stackoverflow.com/questions/41510880/whats-the-difference-between-parsevalue-and-parseliteral-in-graphqlscalartype

on the graphql layer, it's likely we have some check that are related to inputs where the scalar is only used as an output. I din't really spend the time digging into this, this felt very low priority to dive into it.